bpf/disk.bpf.c

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "vmlinux.h"
+#include <linux/limits.h>
+#include <bpf/bpf_helpers.h>       /* most used helpers: SEC, __always_inline, etc */
+#include <bpf/bpf_core_read.h>     /* for BPF CO-RE helpers */
+#include <bpf/bpf_tracing.h>       /* for getting kprobe arguments */
+
+#include "helpers.h"
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+struct val_t {
+    u64 pid;
+    char comm[TASK_COMM_LEN];
+    const char *fname;
+    int flags;
+};
+
+struct data_t {
+    u64 cgroup;
+    u64 pid;
+    int ret;
+    char comm[TASK_COMM_LEN];
+    char fname[NAME_MAX];
+    int flags;
+};
+
+BPF_HASH(infotmp, u64, struct val_t, 8192);
```

Comment:
Here (and in `network.bpf.c`) can you make the hash size a constant and explain why you picked the size?

Reply:
I think that's what bcc's picked as the default size :)